MISampleApp/MISampleWeb/ServiceEventSource.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
```

Comment:
Is this file really necessary? IMO the point of a sample is a minimal but complete-for-purpose exemplification of using a given feature. Everything else just adds noise.